### PR TITLE
[docs] Fixed wrong description concerning passphrase

### PR DESCRIPTION
**Pull request by ethouris**

### CODE REVIEW

**Review by maxsharabayko — APPROVED** [review]

File: docs/APISocketOptions.md [hunk 1]
```diff
@@ -867,20 +867,24 @@ For details, see [Packet Filtering & FEC](packet-filtering-and-fec.md).
...
-(Password-Based Key Derivation Function 2). It is used on the receiver only if
-the received data is encrypted.
+(Password-Based Key Derivation Function 2).
+
+When a socket with configured passphrase is being connected, the peer must
+have the same password set, or otherwise the connection is rejected. This
+behavior can be changed by [`SRTO_ENFORCEDENCRYPTION`](#SRTO_ENFORCEDENCRYPTION).
 
 Note that since the introduction of bidirectional support, there's only one
 initial SEK to encrypt the stream (new keys after refreshing will be updated
```
Comment (jeandube) [inline docs/APISocketOptions.md:883]: This is not part of the PR changes but I have been surprise by this 'SEK' popping here. I don't see it defined around while the KEK is written in full word two paragraphs before. Is there a lexicon we can link to.

Comment (maxsharabayko) [reply docs/APISocketOptions.md:883]: We may link to the [Encryption section](https://tools.ietf.org/html/draft-sharabayko-mops-srt-01#section-5) of the RFC.
This information about SEK is maybe extra in this document: too detailed. I mean, "there's only one
initial SEK to encrypt...".

Comment (ethouris) [reply docs/APISocketOptions.md:883]: I'd simply replace it with "encryption key".

File: docs/APISocketOptions.md [hunk 2]
```diff
@@ -867,20 +867,24 @@ For details, see [Packet Filtering & FEC](packet-filtering-and-fec.md).
 | `SRTO_PASSPHRASE`    | 0.0.0 | pre     | `string`   |         | ""       |[10..79]| W   | GSD    |
 
 Sets the passphrase for encryption. This enables encryption on this party (or
-disables it, if an empty passphrase is passed).
+disables it, if an empty passphrase is passed). The password must be minimum
+10 and maximum 79 characters long. If an empty password is specified (default),
+the encryption is disabled.
```
Comment (stevomatthews) [inline docs/APISocketOptions.md:872]: The sentence "If an empty password is specified (default),
the encryption is disabled." is redundant (see line 870)

File: docs/APISocketOptions.md [hunk 3]
```diff
@@ -867,20 +867,24 @@ For details, see [Packet Filtering & FEC](packet-filtering-and-fec.md).
...
 
 The passphrase is the shared secret between the sender and the receiver. It is
 used to generate the Key Encrypting Key using [PBKDF2](http://en.wikipedia.org/wiki/PBKDF2)
-(Password-Based Key Derivation Function 2). It is used on the receiver only if
-the received data is encrypted.
+(Password-Based Key Derivation Function 2).
+
+When a socket with configured passphrase is being connected, the peer must
+have the same password set, or otherwise the connection is rejected. This
+behavior can be changed by [`SRTO_ENFORCEDENCRYPTION`](#SRTO_ENFORCEDENCRYPTION).
```
Comment (stevomatthews) [inline docs/APISocketOptions.md:880]: When a socket with configured passphrase is being connected, the peer must
have the same password set, or the connection is rejected. This
behavior can be changed by [`SRTO_ENFORCEDENCRYPTION`](#SRTO_ENFORCEDENCRYPTION).

**Review by stevomatthews — COMMENTED** [review]
Minor edits

File: docs/APISocketOptions.md (hunk 2 above)
Comment (ethouris) [reply docs/APISocketOptions.md:872]: Right. How I overlooked it. Will be removed.
